index.html

```diff
@@ -196,7 +196,75 @@ <h3 class="skill-header">Skills</h3>
                     </li> 
                 </ul>
             </section>            
-        </div>        
+        </div>  
+       
+        <footer>
+            <section id="contact-form-section">
+                <div>
+                    <h2 class='contact-form-header' >I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+                </div>
+               <form id="contact-form" action="https://formspree.io/f/xvolvkdz"
+                     method="POST" 
+                >
+                   <input 
+                        class="text-input" 
+                        type="text"  
+                        id="visitors-name"                       
+                        placeholder="Full name"
+                        maxlength="30"
+                        required
+                    >
+                   <input 
+                        class="text-input" 
+                        type="email" 
+                        name="email" 
+                        id="visiotors-email"
+                        placeholder="Email address"                      
+                        required
+                    >
+                    <textarea 
+                        name="contact-text" 
+                        id="contact-text-area" 
+                        cols="30" rows="10" 
+                        maxlength="500" 
+                        required
+                        >Enter Text Here</textarea>
```

Comment:
For a better user experience, and because it was asked specifically in the project requirements, is it better to leave the value of `textarea` empty and give it a placeholder.
Screenshot from project requirements:
![image](https://user-images.githubusercontent.com/83596135/165329807-576a4fd0-3ecc-477a-9b9c-6f7f65506f37.png)

- [ ] Please give `textarea` a placeholder and remove the value "Your message" for a better UX;